doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -32,16 +32,20 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
   
 ## Status
 
-This is a work in progress and is changing on a daily basis.
-Please send your comments to fiware-ngsi@lists.fiware.org.
-You can trace the discussions checking the archives of the mailing list:
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is a release candidate of the NGSIv2 API specification. By "release candidate"
+we mean that the specification is quite stable, but changes may occur with regard to new release
+candidates or the final version. In particular changes may be of two types:
 
-In addition, note that a list of currently open discussions is available at
+* Extensions to the functionality currently specified by this document. Note that in this case
+  there isn't any risk of breaking backward compatibility on existing software implementations.
+* Slight modifications in the functionality currently specified by this document, as a consequence
+  of ongoing discussions. Backward compatibility will be taken into account in this case, trying
+  to minimize the impact on existing software implementations. In particular, only completely
+  justified changes impacting backward compatibility will be allowed and "matter of taste"
+  changes will not be allowed.
```

Comment:
I think is a good idea to provide information to potential API users about the kind of changes they can expect over this release candidate.